fossology/fossology
# feat(scanoss-agent): Initial version of scanoss agent for Fossology

This PR relates to Add SCANOSS Agent #2166
The agent detects licenses by querying file info from osskb DB. This agent fingerprints all the files related to the upload and query information about that file. Only licenses are kept, other information is discarded. The scan results are placed on license_file table.

## Description

This agent fingerprints all the files related to the upload and query information about that file. Only licenses are kept, other information is discarded. The scan results are placed on license_file table.

### Changes

This is the initial commit that only report license information. Other useful information such as copyright and version could be included

## How to test
Just do an upload and check on "SCANOSS Toolkit" from optional analysis

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2167"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
